### PR TITLE
styling done

### PR DESCRIPTION
**Pull request by Luckyaremu**
- styling done
- conflicts resolved
- linters error done

### CODE REVIEW

**Review by Forison — CHANGES_REQUESTED** [review]
## Status:: Changes Required❌
# 🙋‍♂️ @Luckyaremu 
#### You have done a good job on this one so far, here are some issue we have to address🤗.
* [ ] Styling is EXACTLY the same as the one provided in the designs 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone5/exactlySameAsDesign.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/45791599/))
- Let us add the add book row under the page just as in the picture of the requirement.
* [ ]  Let not forget to add an image of the design in the readme and make sure the live links, works. use `git push heroku milestone-5:master` to push to heroku from this branch.
### That is it kindly make these changes and submit a re-review👨‍🔧.
#### Happy coding👨‍💻
Addo Forison

File: src/components/App.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-case-declarations */
```
Comment (Forison) [inline src/components/App.js]: - Do not make changes to eslint.
> note eslint configs should never be modified

File: src/containers/BooksList.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-case-declarations */
```
Comment (Forison) [inline src/containers/BooksList.js]: - Do not make changes to eslint.
> note eslint configs should never be modified

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Luckyaremu :clap: 

**Great work making `almost` all the necessary corrections from the previous review.**

# Status: Changes Required! ♻️ 

1. The live link is broken;
image

2. I saw just one commit; which you removed the eslint inline comment as you were told by @Forison, kindly attend to the changes mentioned by @Forison .

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by jofer86 — APPROVED** [review]
## `const PROJECT_STATUS = "APPROVED"` 🚀 🚀 🚀 
## :wave: :wave: Hello! I'm Jorge, and  I will be conducting this code review. Let me get a closer look 🔎 🔍 🔎 🔍 🕵🏼 
- It seems like all of the required changes were met, great job!!! You might move along now. Milestone complete!!
--------------------------------------------------------------------
**Rock on**  🤘🏼 
**_Jorge Fernando Rincón Figueroa._** 
Technical Support Engineer.

**Pull request merged**